client/components/theme/index.jsx

```diff
@@ -112,8 +112,8 @@ export class Theme extends Component {
 
 	isBeginnerTheme() {
 		const { theme } = this.props;
-		const skillLevels = get( theme, [ 'taxonomies', 'theme_skill-level' ] );
-		return some( skillLevels, { slug: 'beginner' } );
+		const skillLevel = get( theme, [ 'taxonomies', 'theme_skill-level' ] );
+		return skillLevel && skillLevel.slug === 'beginner';
```

Comment:
I think this might have been correct the way it was. Can't confirm of course because we can't find a theme that actually includes the `theme_skill-level` taxonomy. But it looks to me like the items in the  theme's `taxonomies` property are arrays. In which case we should use something like `_.some()` unless I'm mistaken?

![Screen Shot on 2020-09-25 at 15:58:05](https://user-images.githubusercontent.com/1500769/94225009-fc5dfb80-ff47-11ea-947b-a1be18520055.png)